client/my-sites/checkout/cart/popover-cart.jsx

```diff
@@ -79,15 +79,13 @@ const PopoverCart = createReactClass( {
 			<div>
 				<CartMessages cart={ cart } selectedSite={ selectedSite } />
 				<div className={ classes }>
-					<button
-						className="cart-toggle-button"
+					<HeaderButton
+						icon="cart"
+						label="Cart"
```

Comment:
This label needs to be wrapped in a translation function.

Reply:
Nice catches!

I thought about that cart count overlapping as well but I kept it that way, as it appeared to overlap in the current implementation w/o the label.

I've added the translation fn and adjusted that counter positioning in c4ea0e8.

Thanks for the review!

Reply:
Updates look good! Hover state still needs some love, though. There are a couple pixels' of white above and below the gray background:

<img width="963" alt="Screen Shot 2019-10-11 at 12 06 58 PM" src="https://user-images.githubusercontent.com/2124984/66666925-c6811380-ec1f-11e9-8de3-fab64edb7479.png">

Reply:
Good eye! Updated in 6ee20f9.